Source/WebCore/dom/ScriptElement.cpp

```diff
@@ -315,7 +314,7 @@ bool ScriptElement::requestClassicScript(const String& sourceURL)
 {
     ASSERT(m_element.isConnected());
     ASSERT(!m_loadableScript);
-    if (!stripLeadingAndTrailingHTMLSpaces(sourceURL).isEmpty()) {
+    if (!StringView(sourceURL).trim(isASCIIWhitespace<UChar>).isEmpty()) {
```

Comment:
I think I'd use `!StringView(sourceURL).isAllSpecialCharacters<isASCIIWhitespace<UChar>>()` for such checks.